vllm/entrypoints/openai/rpc/client.py

```diff
@@ -335,7 +335,10 @@ async def _is_tracing_enabled_rpc(self) -> bool:
 
     async def abort(self, request_id: str):
         """Send an ABORT_REQUEST signal to the RPC Server"""
-        with suppress(RPCClientClosedError):
+
+        # Suppress timeouts as well- if the server is busy and does not ack in
```

Comment:
I think we should have a very detailed comment here about why this is okay, including the notes from slack